### PR TITLE
Add > Steampipe Cloud > Integrations Doc > HeidiSQL

### PR DESCRIPTION
**Pull request by RahulSrivastav14**

### CODE REVIEW

File: docs/cloud/integrations/heidisql.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  Port:               9193
+  Database:           steampipe
+  User:               steampipe
+  Password:           99**_****_**8c
+  Connection string:  postgres://steampipe:99**_****_**8c@localhost:9193/steampipe
+  ```
+
+## Getting started
+
+[HeidiSQL](https://www.heidisql.com/download.php) is a free to use program that runs only on a Windows desktop. In this example we will create a Steampipe Cloud connection with HeidiSQL and query the top stories from Hacker News.
```
Comment (judell) [inline docs/cloud/integrations/heidisql.md:36]:
```suggestion
[HeidiSQL](https://www.heidisql.com/download.php) is a program that runs on Windows. In this example we will create a Steampipe Cloud connection with HeidiSQL and query the top stories from Hacker News.
```

File: docs/cloud/integrations/heidisql.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<img src="/images/docs/cloud/heidisql-connection-success.png" />
+</div>
+
+Once you've connected to Steampipe Cloud, you can access the Hacker News plugin and its tables from the Database explorer.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"50%"}}>
+<img src="/images/docs/cloud/heidisql-database-explorer.png" />
+</div>
+
+Here we select the `hackernews_top` table. HeidiSQL displays the table's schema and previews the returned data. You can drag the columns to re-arrange the data and also save it to CSV, HTML, XML, LaTeX, Wiki markup or PHP files.
```
Comment (judell) [inline docs/cloud/integrations/heidisql.md:50]:
```suggestion
Here we select the `hackernews_top` table. HeidiSQL displays the table's schema and previews the returned data. You can drag the columns to rearrange the data, then save to CSV, HTML, XML, LaTeX, Wiki markup or PHP.
```

Comment (RahulSrivastav14) [reply docs/cloud/integrations/heidisql.md:50]: Updated the suggested text

File: docs/cloud/integrations/heidisql.md (hunk 1 above)
Comment (RahulSrivastav14) [reply docs/cloud/integrations/heidisql.md:36]: Updated the suggested text

**Review by judell — APPROVED** [review]

**Pull request merged**